website/server/controllers/api-v3/auth.js

```diff
@@ -162,8 +162,10 @@ api.registerLocal = {
 };
 
 function _loginRes (user, req, res) {
+  var newUser = false;
```

Comment:
Actually you don't need to create a new car at all. You can just pass user.newUser || false